1-js/05-data-types/01-primitives-methods/article.md

```diff
@@ -61,15 +61,15 @@ let str = "Hello";
 alert( str.toUpperCase() ); // HELLO
 ```
 
-Simple, right? And here's what actually happens in `str.toUpperCase()`:
+Simple, right? Here's what actually happens in `str.toUpperCase()`:
 
-1. The string `str` is a primitive. So in the moment of accessing its property a special object is created that both knows the value of the string and has useful methods, like `toUpperCase()`.
+1. The string `str` is a primitive. So in the moment of accessing its property, a special object is created that knows the value of the string, and has useful methods, like `toUpperCase()`.
```

Comment:
"…a special object is created that knows the value of the string, and has useful methods…" -- is the comma really needed here (before "and")?

Reply:
See my other comment about the Oxford comma.

Reply:
@gwooly please remove Oxford comma, not used in other articles.